dts-toip/backup-mongodb/dev-values.yaml

```diff
@@ -0,0 +1,130 @@
+# Default values for backup-storage.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+image:
+  repository: i5okie/backup-mongo
```

Comment:
Should we have a bcgov image listed here instead? Same for other image references.

Reply:
Certainly. Maybe when this gets merged, and an "official" image can be built?
https://github.com/BCDevOps/backup-container/pull/116